src/components/forms/AcademicGrantsForm.tsx

```diff
@@ -0,0 +1,594 @@
+      .catch(err => console.error('There has been a problem with your operation: ', err.message));
+  };
+
+  return (
+    <Stack
+      w='100%'
+      bgGradient='linear(to-br, brand.newsletter.bgGradient.start 10%, brand.newsletter.bgGradient.end 100%)'
+      px={{ base: 5, md: 12 }}
+      pt={{ base: 8, md: 12 }}
+      pb={{ base: 20, md: 16 }}
+      borderRadius={{ md: '10px' }}
+    >
+      <FormProvider {...methods}>
+        <Flex
+          as='form'
+          id='academic-grants-form'
+          onSubmit={handleSubmit(onSubmit)}
+          noValidate
+          direction='column'
+          gap={8}
+        >
+          <Flex direction='column'>
+            <Flex direction={{ base: 'column', md: 'row' }} mb={3}>
+              <TextField
+                id='firstName'
+                label='First name'
+                isRequired
+                mr={{ md: 12 }}
+                mb={{ base: 8, md: 0 }}
+              />
+
+              <TextField id='lastName' label='Last name' isRequired />
+            </Flex>
+
+            {!errors?.firstName && !errors?.lastName && (
+              <PageText as='small' fontSize='helpText' color='brand.helpText'>
+                Who is the point of contact for the application?
+              </PageText>
+            )}
+          </Flex>
+
+          <TextField id='email' label='Email' isRequired />
+
+          <Controller
+            name='POCisAuthorisedSignatory'
+            control={control}
+            render={({ field: { onChange, value } }) => (
+              <Field
+                id='POCisAuthorisedSignatory'
+                label='Is the point of contact also the authorised signatory?'
+                isRequired
+              >
+                <RadioGroup
+                  id='POCisAuthorisedSignatory'
+                  onChange={value => onChange(value === 'Yes')}
+                  value={value ? 'Yes' : 'No'}
+                  fontSize='input'
+                  colorScheme='white'
+                  mt={4}
+                >
+                  <Stack direction='row'>
+                    <Radio
+                      id='POCisAuthorisedSignatory-yes'
+                      size='lg'
+                      name='POCisAuthorisedSignatory'
+                      value='Yes'
+                      defaultChecked
+                      mr={8}
+                    >
+                      <PageText fontSize='input'>Yes</PageText>
+                    </Radio>
+
+                    <Radio
+                      id='POCisAuthorisedSignatory-no'
+                      size='lg'
+                      name='POCisAuthorisedSignatory'
+                      value='No'
+                    >
+                      <PageText fontSize='input'>No</PageText>
+                    </Radio>
+                  </Stack>
+                </RadioGroup>
+              </Field>
+            )}
+          />
+
+          <Box display={notAuthorisedSignatory ? 'block' : 'none'}>
```

Comment:
this field is missing the error PageText when not completed

Reply:
This is a limitation of Zod, but the validation is done. This is related to the `.refine' method which is validated after the other fields in the schema. If you fill out the other fields and then you submit again, you should see this error popup. I double-checked it on my end and it shows the error.